### DISCUSSION TITLE
Site asks me to 'enable javascript and cookies'

### DISCUSSION THREAD

**Discussion by johnsturgeon** [FreshRSS/FreshRSS#6546] [Q&A]
I'm trying to add the following feed:

https://www.livingonthecotedazur.com/category/blog/feed/

and I cannot add it.   Any help / tips or pointers would be great!

Digging a bit deeper I got:
`<span id="challenge-error-text">Enable JavaScript and cookies to continue</span>`

I have checked 'accept cookies' but 'enable javascript' is a new one to me

**Comment by Alkarex** [comment]
```console
$ curl -v 'https://www.livingonthecotedazur.com/category/blog/feed/'
...
< server: cloudflare
...
```

It is yet again an instance of Cloudflare breaking the Web. Same than e.g. https://github.com/FreshRSS/FreshRSS/discussions/6520#discussioncomment-9617541 and https://github.com/search?q=repo%3AFreshRSS%2FFreshRSS+Cloudflare&type=issues

https://github.com/FreshRSS/FreshRSS/issues/4323
Help welcome to centralise an FAQ on the topic.

**Reply by math-GH** [reply]
FAQ is started here: #6287
